packages/libs/react-ui/src/components/Notification/Notification.css.ts

```diff
@@ -1,16 +1,14 @@
 import { sprinkles } from '@theme/sprinkles.css';
-import type { ColorType } from '@theme/vars.css';
 import { vars } from '@theme/vars.css';
 import { style, styleVariants } from '@vanilla-extract/css';
 
-const colors: Record<ColorType, ColorType> = {
+export type NotificationColors = 'info' | 'positive' | 'warning' | 'negative' | 'primary';
+const colors: Record<NotificationColors, NotificationColors> = {
```

Comment:
I think it's better to use `Omit<ColorType, "secondary" | "tertiary">` since this ties this type to the original